### DISCUSSION TITLE
CSS styles

### DISCUSSION THREAD

**Discussion by mikekenyon99** [totaldebug/atomic-calendar-revive#1186] [Support] [answered]
Hi,

I have looked through the documentation but I cannot see which CSS Styles I can set with Card_Mod on the atomic-calendar? Am I missing something really obvious? Or is there a standard list on the net somewhere?

Thanks

Mike

**Comment by marksie1988** [comment, marked as answer]
The best way is to use developer tools in your browser.

Press f12 then you should have an option to select an element, use that and you can select the area you want to change, it will take you to the code which you can the see the relevant class.

**Comment by mikekenyon99** [comment]
Thanks!
